tests/integration/tickstore/test_toplevel.py

```diff
@@ -88,6 +88,28 @@ def test_should_return_data_when_date_range_spans_libraries(toplevel_tickstore,
     assert_frame_equal(expected_df, res.tz_convert(mktz('Europe/London')))
 
 
+def test_should_return_data_when_date_range_spans_libraries_even_if_one_returns_nothing(toplevel_tickstore, arctic):
+    arctic.initialize_library('FEED_2010.LEVEL1', tickstore.TICK_STORE_TYPE)
+    arctic.initialize_library('FEED_2011.LEVEL1', tickstore.TICK_STORE_TYPE)
+    tickstore_2010 = arctic['FEED_2010.LEVEL1']
+    tickstore_2011 = arctic['FEED_2011.LEVEL1']
+    toplevel_tickstore._collection.insert_one({'start': dt(2010, 1, 1),
```

Comment:
Can we not use `TopLevelTickStore.add` here rather than crafting a mapping by hand?